pypeit/reduce.py

```diff
@@ -1138,7 +1237,7 @@ def local_skysub_extract(self, global_sky, sobjs,
         return self.skymodel, self.objmodel, self.ivarmodel, self.outmask, self.sobjs
 
 
-class IFUReduce(Reduce):
+class IFUReduce(MultiSlitReduce, Reduce):
```

Comment:
Didn't know you can do that.  Do you need to?
ie.  isn't just `MultiSlitReduce` enough?

Reply:
Yep, both are needed (and order is important). When writing this, I read about multiple inheritance and method resolution order, and decided this order was needed and both classes needed to be specified. I've forgotten why...